bach/bach/series/series.py

```diff
@@ -1215,6 +1215,32 @@ def window_nth_value(self, n: int, window: WrappedWindow = None):
             self.dtype
         )
 
+    def append(
+        self,
+        other: Union['Series', List['Series']],
+        ignore_index: bool = False,
+    ) -> 'Series':
+        """
+        Append rows of other series to the caller series.
+
+        :param other: objects to be added
+        :param ignore_index: if true, drops indexes of all objects to be appended
+
+        :return: a new series with all rows from appended other.
+        """
+        from bach.concat import SeriesConcatOperation
+        if not other:
+            raise ValueError('no series to append.')
```

Comment:
```suggestion
            return self
```
Seems that this is what pandas does, and I think it is a bit more logical too. saves our end users an edge-case that they have to handle

The docstring should then be something like
`:return: a new series with all rows from appended other or self if other is empty`